gm4_mountaineering/data/gm4_mountaineering/functions/pole_effects/poles_jump_boost.mcfunction

```diff
@@ -0,0 +1,15 @@
+# Give jump boost effects when holding poles, with a cooldown
+# @s = player holding poles
+# at @s
+# run from poles_equipped
+
+# Add jump boost effect (1 regularly, 2 if having an upwards velocity)
+execute if score @s gm4_mountaineering_poles_jump_boost matches ..0 if score @s gm4_mountaineering_vy matches 2.. run effect give @s jump_boost 1 1 true
+execute if score @s gm4_mountaineering_poles_jump_boost matches ..0 if score @s gm4_mountaineering_vy matches ..0 run effect give @s jump_boost 1 0 true
```

Comment:
jump boost level 1 feels rather inconsequential, I would recommend changing the base level to 2, then use 3-4 for the increased one. You might also want to reduce the `gm4_mountaineering_vy` threshold a bit as it felt really difficult to reach the higher-level jump boost

Reply:
Reducing the threshold would be a bit difficult to do without reworking the whole vy system (since it only works with integers). So I've done this instead:
- if you wear skis, you get jump boost 1 (same as current system); jumping while wearing skis is a bit difficult
- if you don't wear skis, you get jump boost 2
- regardless of whether you wear skis or not, you get jump boost 3 when moving uphill; this much larger jump boost (I think) should compensate for the fact that you don't get it all the time